### PR TITLE
Amazon SQS / AWSO 2.6.0 (early Apr 2023)

### PR DESCRIPTION
**Pull request by kimsauce**
- Adds a new AWSO feature being released in late Jan
- @JKashyap96 submitted this PR originally and handed off to N. Pande

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1203622662622977

### CODE REVIEW

File: docs/observability/aws/integrations/amazon-sqs.md [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+| where (tolowercase(queuename) matches tolowercase("*")) or isBlank(queuename)
+| if (isBlank(recipient_account_id), accountid, recipient_account_id) as accountid
+| if (isEmpty(error_code), "Success", "Failure") as event_status
+| count as event_count by username
+| top 10 username by event_count, username asc
+```
+
+## Field in Field Schema
+
+Login to Sumo Logic,  go to Manage Data > Logs > Fields. Search for the “queuename” field. If not present, create it. Learn how to create and manage fields [here](https://help.sumologic.com/Manage/Fields#manage-fields).
```
Comment (stacykor) [inline docs/observability/aws/integrations/amazon-sqs.md:96]: Login to Sumo Logic, go to **Manage Data** > **Logs** > **Fields**. Search for the “queuename” field. If not present, create it. Learn how to create and manage fields [here](/docs/manage/fields/#manage-fields).

**Review by stacykor — CHANGES_REQUESTED** [review]
Two fixes:

1) the manage fields reference looks off, suggested a local file path.
2) there's an extra space

File: docs/integrations/amazon-aws/sqs.md [hunk 2]
```diff
@@ -135,6 +135,7 @@ Namespace for **Amazon SQS** Service is **AWS/SQS**
 Login to Sumo Logic,  go to Manage Data > Logs > Fields. Search for the “queuename” field. If not present, create it. Learn how to create and manage fields [here](https://help.sumologic.com/Manage/Fields#manage-fields).
```
Comment (kimsauce) [inline docs/integrations/amazon-aws/sqs.md:135]:
```suggestion
Log in to Sumo Logic, go to **Manage Data** > **Logs** > **Fields**. Search for the “queuename” field. If not present, create it. Learn how to create and manage fields [here](https://help.sumologic.com/Manage/Fields#manage-fields).
```

File: docs/observability/aws/integrations/amazon-sqs.md (hunk 1 above)
Comment (kimsauce) [inline docs/observability/aws/integrations/amazon-sqs.md:96]:
```suggestion
Log in to Sumo Logic, go to **Manage Data** > **Logs** > **Fields**. Search for the “queuename” field. If not present, create it. Learn how to create and manage fields [here](https://help.sumologic.com/Manage/Fields#manage-fields).
```

File: docs/observability/aws/integrations/amazon-sqs.md [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+
+## Field Extraction Rule(s)
+
+Create a Field Extraction Rule for CloudTrail Logs. Learn how to create a Field Extraction Rule [here](https://help.sumologic.com/Manage/Field-Extractions/Create-a-Field-Extraction-Rule).
```
Comment (kimsauce) [inline docs/observability/aws/integrations/amazon-sqs.md:100]: This is a link to our legacy docs -- replace with current version

File: docs/integrations/amazon-aws/sqs.md [hunk 4]
```diff
@@ -132,9 +132,10 @@ Namespace for **Amazon SQS** Service is **AWS/SQS**
 
 ## Field in Field Schema
 
-Login to Sumo Logic,  go to Manage Data > Logs > Fields. Search for the “queuename” field. If not present, create it. Learn how to create and manage fields [here](https://help.sumologic.com/Manage/Fields#manage-fields).
+Log in to Sumo Logic, go to **Manage Data** > **Logs** > **Fields**. Search for the “queuename” field. If not present, create it. Learn how to create and manage fields [here](https://help.sumologic.com/Manage/Fields#manage-fields).
```
Comment (kimsauce) [inline docs/integrations/amazon-aws/sqs.md:135]: This is a link to our legacy docs -- replace with current version

File: docs/integrations/amazon-aws/sqs.md [hunk 5]
```diff
@@ -132,9 +132,10 @@ Namespace for **Amazon SQS** Service is **AWS/SQS**
...
 
 ## Field Extraction Rule(s)
+
 Create a Field Extraction Rule for CloudTrail Logs. Learn how to create a Field Extraction Rule [here](https://help.sumologic.com/Manage/Field-Extractions/Create-a-Field-Extraction-Rule).
```
Comment (kimsauce) [inline docs/integrations/amazon-aws/sqs.md:139]: This is a link to our legacy docs -- replace with current version

**Comment by kimsauce** [comment]
Hi @JKashyap96 and @zjiawei3 , I've assigned your PR back to you -- @stacykor and I have added some edits.

**Comment by zjiawei3** [comment]
> Hi @JKashyap96 and @zjiawei3 , I've assigned your PR back to you -- @stacykor and I have added some edits.

Hi @kimsauce , I dont think I have any context about this PR. Did you tag the wrong person?

File: docs/integrations/amazon-aws/sqs.md (hunk 4 above)
Comment (kimsauce) [inline docs/integrations/amazon-aws/sqs.md:135]:
```suggestion
Log in to Sumo Logic, go to **Manage Data** > **Logs** > **Fields**. Search for the “queuename” field. If not present, create it. Learn how to create and manage fields [here](/docs/manage/fields/#manage-fields).
```

File: docs/observability/aws/integrations/amazon-sqs.md [hunk 6]
```diff
@@ -0,0 +1,169 @@
...
+| where (tolowercase(queuename) matches tolowercase("*")) or isBlank(queuename)
+| if (isBlank(recipient_account_id), accountid, recipient_account_id) as accountid
+| if (isEmpty(error_code), "Success", "Failure") as event_status
+| count as event_count by username
+| top 10 username by event_count, username asc
+```
+
+## Field in Field Schema
+
+Log in to Sumo Logic, go to **Manage Data** > **Logs** > **Fields**. Search for the “queuename” field. If not present, create it. Learn how to create and manage fields [here](https://help.sumologic.com/Manage/Fields#manage-fields).
```
Comment (kimsauce) [inline docs/observability/aws/integrations/amazon-sqs.md:96]:
```suggestion
Log in to Sumo Logic, go to **Manage Data** > **Logs** > **Fields**. Search for the “queuename” field. If not present, create it. Learn how to create and manage fields [here](/docs/manage/fields/#manage-fields).
```

File: docs/observability/aws/integrations/amazon-sqs.md [hunk 7]
```diff
@@ -0,0 +1,169 @@
...
+| top 10 username by event_count, username asc
+```
+
+## Field in Field Schema
+
+Log in to Sumo Logic, go to **Manage Data** > **Logs** > **Fields**. Search for the “queuename” field. If not present, create it. Learn how to create and manage fields [here](/docs/manage/fields/#manage-fields).
+
+## Field Extraction Rule(s)
+
+Create a Field Extraction Rule for CloudTrail Logs. Learn how to create a Field Extraction Rule [here](https://help.sumologic.com/Manage/Field-Extractions/Create-a-Field-Extraction-Rule).
```
Comment (kimsauce) [inline docs/observability/aws/integrations/amazon-sqs.md:100]:
```suggestion
Create a Field Extraction Rule for CloudTrail Logs. Learn how to create a Field Extraction Rule [here](/docs/manage/field-extractions/create-field-extraction-rule).
```

File: docs/integrations/amazon-aws/sqs.md [hunk 8]
```diff
@@ -132,9 +132,10 @@ Namespace for **Amazon SQS** Service is **AWS/SQS**
 
 ## Field in Field Schema
 
-Login to Sumo Logic,  go to Manage Data > Logs > Fields. Search for the “queuename” field. If not present, create it. Learn how to create and manage fields [here](https://help.sumologic.com/Manage/Fields#manage-fields).
+Log in to Sumo Logic, go to **Manage Data** > **Logs** > **Fields**. Search for the “queuename” field. If not present, create it. Learn how to create and manage fields [here](/docs/manage/fields/#manage-fields).
 
 ## Field Extraction Rule(s)
+
 Create a Field Extraction Rule for CloudTrail Logs. Learn how to create a Field Extraction Rule [here](https://help.sumologic.com/Manage/Field-Extractions/Create-a-Field-Extraction-Rule).
```
Comment (kimsauce) [inline docs/integrations/amazon-aws/sqs.md:139]:
```suggestion
Create a Field Extraction Rule for CloudTrail Logs. Learn how to create a Field Extraction Rule [here](/docs/manage/field-extractions/create-field-extraction-rule).
```

File: docs/observability/aws/integrations/amazon-sqs.md [hunk 9]
```diff
@@ -0,0 +1,169 @@
...
+}
+```
+
+
+
+### Sample Query
+
+**Messages Received (Metric based)**:
+
+```
```
Comment (kimsauce) [inline docs/observability/aws/integrations/amazon-sqs.md:72]:
```suggestion
```sql
```

**Comment by kimsauce** [comment]
Hi @kimsauce , I dont think I have any context about this PR. Did you tag the wrong person?

Ah, sorry! Please ignore.

File: docs/observability/aws/integrations/amazon-sqs.md (hunk 9 above)
Comment (JKashyap96) [reply docs/observability/aws/integrations/amazon-sqs.md:72]: Looks good how it is.

**Review by JKashyap96 — APPROVED** [review]
LGTM, will merge this when we release AWSO 2.6 next quarter

File: docs/observability/aws/faq.md [hunk 10]
```diff
@@ -22,11 +22,11 @@ Our AWS Observability solution enables you to view your entire AWS environment i
 
 * **AWS Observability Solution** - Sumo Logic’s AWS Observability solution pulls in data across key AWS services and accounts to give a unified view of AWS environments. Easily navigate from overview dashboards into account, region, availability zone, or service specific views. Intuitive navigation ensures teams can quickly resolve issues, minimize downtime, and improve system availability. See [About AWS Observability](docs/observability/aws/about.md).
 
-See [AWS Observability Apps](/docs/observability/aws/integrations) for detailed information on all supported apps including Application, Classic, and Network Load Balancer, DynamoDB, EC2, RDS, API Gateway, ECS, ElastiCache, Lambda, and SNS.
+See [AWS Observability Apps](/docs/observability/aws/integrations) for detailed information on all supported apps including Application, Classic, and Network Load Balancer, DynamoDB, EC2, RDS, API Gateway, ECS, ElastiCache, Lambda, SNS and SQS.
```
Comment (kimsauce) [inline docs/observability/aws/faq.md:25]:
```suggestion
See [AWS Observability Apps](/docs/observability/aws/integrations) for detailed information on all supported apps including Application, Classic, and Network Load Balancer, DynamoDB, EC2, RDS, API Gateway, ECS, ElastiCache, Lambda, SNS, and SQS.
```

**Comment by kimsauce** [comment]
@JKashyap96 checking in - do you have an update when we can merge this?

**Comment by JKashyap96** [comment]
@kimsauce Will find out and get back to you.

**Comment by JKashyap96** [comment]
@kimsauce We are planning to release AWSO 2.6 by the end of January 2023, so would need to be merged by then. We might have more commits to this PR later as well.

**Comment by kimsauce** [comment]
@JKashyap96 , is this ready to publish? cc: @stacykor @nmohantyctr

**Comment by kimsauce** [comment]
Spoke with Jayanta, who's handing this off to NPande. @nmohantyctr reassigning to you as it's App-related and created an Asana ticket for you.

**Comment by kimsauce** [comment]
> Spoke with Jayanta, who's handing this off to NPande. @nmohantyctr reassigning to you as it's App-related and created an Asana ticket for you. cc @stacykor

@nmohantyctr did you get a chance to check with Nitin on the release date? cc: @stacykor

**Comment by nmohantyctr** [comment]
Hello, @kimsauce Spoke to Nitin, Jayanta, and Himanshu Sharma about this feature. All the changes are in place. However, the release (for AWSO 2.6) is now currently due around the end of February.
cc @stacykor @JKashyap96

**Pull request merged**